src/services/upper-bar-manager/ui/dropdown-bar-item/dropdown-bar-item.tsx

```diff
@@ -58,6 +59,8 @@ const DropdownBarItem = ({ displayName, text, icon, onClick, onDropdownClick, to
     textElement
   );
 
+  const textString = typeof text === 'string' ? text : renderToString(text);
```

Comment:
what does renderToString accept ? ONLY preact components ?

Reply:
renderToString can work with the following: 
const result1 = renderToString(<MyComponent />); // Works with Preact component
const result2 = renderToString(<div>Plain HTML</div>); // Works with JSX (plain HTML)
const result3 = renderToString(<Fragment><span>Item 1</span><span>Item 2</span></Fragment>); // Works with Fragments

Reply:
what happens if it accepts a string, function, or object ?

Reply:
- String and JSX Elements are valid arguments and will be rendered correctly.

- It works fine it seems with our localization wrappers components as well, that previously returned [Object object] such as <Text> and <Localizer> components.

- As for function, the case is covered before the text prop get's passed as an argument, here: https://github.com/kaltura/playkit-js-ui-managers/blob/master/src/services/upper-bar-manager/ui/dropdown-bar/dropdown-bar.component.tsx#L43

Reply:
alright, sounds good